### ISSUE TITLE
Selector ID and Class case naming

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#1356]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Using the `"selector-type-case": "lower",` I expected `.fooBar { color: pink; }` to throw a warning.

> Which rule, if any, is this issue related to?

Related to  `selector-type-case`

> What CSS is needed to reproduce this issue?

A few quick examples off the top of my head:
```css
.Foo,
.fooBar,
#Foo_Bar,
#foo_bar {
  color: pink;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "selector-type-case": "lower",
  }
}
```

> Which version of stylelint are you using?

`6.5.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint --config myconfig *.css`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

Warnings to be flagged for the use of snake_case and camelCase selector names

> What actually happened (e.g. what warnings or errors you are getting)?

No warnings were flagged

**Comment by jeddy3** [comment]
Those are *class* and *id* selectors, not *type* selectors ([ref](https://drafts.csswg.org/selectors-4/))

You'll want to use `selector-class-pattern` and `selector-id-pattern` if you want enforce to lowercase *id* and *class* selectors.

**Comment by ntwb** [comment]
Ah, thanks @jeddy3, I'll go test some regex's  👍
